packages/server/lib/cloud/protocol.ts

```diff
@@ -58,39 +55,46 @@ class ProtocolManagerImpl implements ProtocolManager {
     this.protocol = await setupProtocol(url)
   }
 
-  async connectToBrowser (options) {
+  async connectToBrowser (cdpClient) {
     debug('connecting to browser for new spec')
 
-    return this.protocol?.connectToBrowser(options)
+    return this.protocol?.connectToBrowser(cdpClient)
   }
 
-  addRunnables (runnables) {
-    this.protocol?.addRunnables(runnables)
+  async addRunnables (runnables) {
```

Comment:
```suggestion
 addRunnables (runnables) {
```

Reply:
We're keeping it. Returning the call to `this.protocol?.addRunnables` will make this async because `this.protocol?.addRunnables` is also async.

Reply:
doesn't that just mean this function returns a `Promise<T>` vs ` Promise<Promise<T>>`? I don't think we need it unless there is some other reason?

Reply:
I don't think its blocking by any means but it might be an unnecessary promise wrapper under the hood.

Reply:
Ahh doy. I see what you mean. I updated this to actually await here. I think that makes things clearer that we're dealing with async calls.